modules/ROOT/pages/salesforce/salesforce-connector-100-upgrade-migrate.adoc

```diff
@@ -167,7 +170,7 @@ The following operations were removed from Salesforce Connector:
  parameter and a Create batch operation
  | Delete multiple | Delete operation
 | Get deleted range | Get deleted objects operation
-| Get updated | Get updated objects operations
+| Get updated | Get updated objects operation
```

Comment:
It would be a lot easier to read narrative if operation names were in title-style capitalization: Get Updated Objects operation. Can we do that?